pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "hatchling.build"
 
 [project]
 name = "helical"
-version = "0.0.1a9"
+version = "0.0.1a13"
 authors = [
```

Comment:
Version is updated here only but needs to be updated in README, docs/index.md and citation.bib to reflect this version